examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                          chains=2, cores=1, init='adapt_diag', random_seed=42)
</pre><p>For <code>v4</code>  the typical variable name would be <code>idata</code>  for the returned <code>InferenceData</code> object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>even if using v3, all notebooks should already use <code>return_inferencedata=True</code>,  and if possible use coords and dims instead of shape.</p>